packages/py-moose-lib/moose_lib/streaming/streaming_function_runner.py

```diff
@@ -167,6 +167,14 @@ def load_streaming_function_dmv2() -> tuple[type, Callable]:
     Raises:
         SystemExit: If module import fails or if no matching transformation is found
     """
+    sys.path.append(function_file_dir)
```

Comment:
my hunch is that we don't need to append to `sys.path` if the `function_file_name` is `"app.main"` - this is hardcoded into `dmv2-serializer.py` anyway.